libcacophony.c

```diff
@@ -0,0 +1,44 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <assert.h>
+
+void write_int(long long x) {
+    printf("%lld", x);
```

Comment:
I think given what we have right now, it's better to add a newline at the end

Reply:
There is write_char(10) :)